free-courses-hi.md

```diff
@@ -12,6 +12,8 @@
 ### Android
 
 * [Android Development Tutorials in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9aiL0kysYlfSOUgY5rNlOhUd) - Harry
+* [Android App Development Tutorial in Hindi](https://www.youtube.com/watch?v=EplH-amHTtE&list=PLUhfM8afLE_Ok-0Lx2v9hfrmbxi3GgsX1) - Neat Roots
```

Comment:
We prefer YouTube URLs to be to the `/playlist` endpoint rather than the `/watch` endpoint. Could you update this so users can get a better overview of the entire course?

We prefer to use the title as defined by the resource, please don't create your own title.

```suggestion
* [Complete Android Development Course in Hindi](https://www.youtube.com/playlist?list=PLUhfM8afLE_Ok-0Lx2v9hfrmbxi3GgsX1) - Neat Roots
```